docs/howtos/extending/magicgui.md

```diff
@@ -129,6 +129,7 @@ as soon as we add the 'camera' image.
 
 ```{code-cell} python
 :tags: [remove-output]
+from magicgui import magicgui
```

Comment:
While we're here, could you move the `import napari` line 135 (new 136) to the top? may as well use proper import sorting.